### ISSUE TITLE
Manifest not setup correctly for the prebuilt appxbundle?

### ISSUE THREAD

**Issue by VioletGiraffe** [microsoft/winget-cli#1018]
# Brief description of your issue
The application name is not specify correctly in the appxbundle hosted in "Releases".
# Steps to reproduce
1. Download Microsoft.DesktopAppInstaller_8wekyb3d8bbwe.appxbundle from Releases (the 1.0 release).
2. Launch it.
3. I'm greeted with this screen, which doesn't look quite right:

![image](https://user-images.githubusercontent.com/1175926/120076736-cb722f80-c0af-11eb-99f4-38f584174685.png)

Expected: a meaningful human-readable name of the application being installed.
The version also doesn't quite match What WinGet displays itself - expected 1.0.***, but show 1.11.***
I guess this isn't actually the version of the WinGet package, so it may be by design, but it's confusing nonetheless.

# Environment

```none
[winget --info]
Windows Package Manager v1.0.11451
Windows: Windows.Desktop v10.0.18363.1379
Package: Microsoft.DesktopAppInstaller v1.11.11451.0
```

**Comment by denelon** [comment]
@VioletGiraffe I think we've seen this before. I believe it is related to the version of the "App Installer" you had on your system prior to installing v1.11.11451. Now that you have a "newer" version of the "App Installer", can you repeat the process to see if the name is properly displayed?

I also gave a [short explanation](https://github.com/microsoft/winget-cli/discussions/988#discussioncomment-798401) of the version differences.

**Comment by VioletGiraffe** [comment]
@denelon If I just run the appxbundle again, I get "App installer is already installed". Which is much better, but still confusing because I thought I'm installing a tool named "WinGet", but I don't see it being installed at any point during the process. But my main nitpick is about the name displayed upon the first installation. It's just a cosmetic issue, of course.

I wanted to uninstall and reinstall WinGet, as if I had the latest version of App Installer at first installation of WinGet, but I don't see a way to uninstall it - it's not registered either in "Add or remove programs", nor in Windows Store - Installed apps.

**Comment by denelon** [comment]
@VioletGiraffe the Windows Package Manager ships inside of the "App Installer". I'm hoping this confusion is resolved when we are able to add this version to the store. At that point, we will add language for "Windows Package Manager" and the "winget" client in search topics as well. We needed to provide sufficient time for enterprise customers to configure and deploy Group Policy before we pushed this to all Windows 10 version 1809 and newer builds.

**Comment by denelon** [comment]
I've marked this as "Area-External". The bug is with "older" versions of the "App Installer". Once we publish this version as the default version in the store for Windows 10 version 1809 and newer, this Bug will be resolved.

**Comment by denelon** [comment]
@VioletGiraffe

We've published the v1.1 [release candidate](https://github.com/microsoft/winget-cli/releases/tag/v1.1.12653) and updated our [troubleshooting guide](https://github.com/microsoft/winget-cli/tree/master/doc/troubleshooting).

Can you confirm if this is still happening?
